src/main/java/org/spburegistry/backend/service/FacultyService.java

```diff
@@ -0,0 +1,14 @@
+package org.spburegistry.backend.service;
+
+import org.spburegistry.backend.dto.FacultyRequestTO;
+import org.spburegistry.backend.dto.FacultyTO;
+import org.spburegistry.backend.entity.Faculty;
+
+public interface FacultyService {
+    Iterable<FacultyTO> findAll();
+
+    FacultyTO findById(long id);
+    Faculty createFaculty(String name, String link);
```

Comment:
You don't use this method as public interface method. remove it from interface and made private (without @Override) in FacultyServiceImpl.